WebRoot/js/zimbraAdmin/dl/view/ZaNewDLXWizard.js

```diff
@@ -1376,7 +1376,7 @@ ZaNewDLXWizard.myXFormModifier = function(xFormObject, entry) {
                                     nowrap: false,
                                     msgName: ZaMsg.LBL_DL_Type,
                                     subLabel: "",
-                                    visibilityChecks: [],
+                                    visibilityChecks: [ZaZimbraAdmin.canCreateGroup],
```

Comment:
can we move this to enableDisableChecks? i guess normally we hide items when we don't have read permissions and we disable items when we don't have write permissions

Reply:
The reason is the same as the above. A field should be hidden if we don't have write permission on New DL wizard.